Code-Hills/codehills-be
# Chore - Configure eslint/prettier and husky pre-commit hook

#### What does this PR do?
> - This PR sets up Eslint, prettier and husky pre-commit hook

#### How should this be manually tested?
> - clone this repository
> - checkout to this branch `ch-pre-commit-eslint`
> - install the dependencies `npm install`
> - Make a commit and the pre-commit script should run `tests` and `lint:fix` before the commit
